### PR TITLE
Disable Saleor_0702 for 3.5 or higher

### PR DESCRIPTION
**Pull request by szczecha**
I want to merge this change because:
- it disables SALEOR_0702 on version 3.5 and higher
- adds SALEOR_0712 for 3.5 and greater

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://marketplace-gray.vercel.app/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7621 Disable Saleor_0702 for 3.5 or higher](https://app.clickup.com/t/2549495/SALEOR-7621)

File: cypress/e2e/configuration/channels/channels.js [hunk 1]
```diff
@@ -86,32 +98,62 @@ describe("Channels", () => {
...
-      getShippingZone(shippingZone.id).then(shippingZoneResp => {
-        const assignedChannel = shippingZoneResp.channels.find(
-          channel => channel.name === randomChannel,
-        );
-        expect(assignedChannel).to.be.ok;
-      });
-    },
-  );
+        const randomChannel = `${channelStartsWith} ${faker.datatype.number()}`;
+        cy.addAliasToGraphRequest("Channels");
```
Comment (eCzerniak) [inline cypress/e2e/configuration/channels/channels.js:110]:
```suggestion

        cy.addAliasToGraphRequest("Channels");
```

File: cypress/e2e/configuration/channels/channels.js [hunk 2]
```diff
@@ -86,32 +98,62 @@ describe("Channels", () => {
...
+  } else {
+    it(
+      "should create channel with shippingZone. TC: SALEOR_0702",
+      { tags: ["@channel", "@allEnv"] },
+      () => {
+        // remove login after fixing SALEOR-3162
+        cy.clearSessionData().loginUserViaRequest();
+
+        const randomChannel = `${channelStartsWith} ${faker.datatype.number()}`;
+        cy.addAliasToGraphRequest("Channels");
```
Comment (eCzerniak) [inline cypress/e2e/configuration/channels/channels.js:138]:
```suggestion

        cy.addAliasToGraphRequest("Channels");
```

**Review by eCzerniak — APPROVED** [review]

File: cypress/e2e/configuration/channels/channels.js [hunk 3]
```diff
@@ -85,8 +97,35 @@ describe("Channels", () => {
...
+      cy.waitForRequestAndCheckIfNoErrors("@Channel");
+      getShippingZone(shippingZone.id).then(shippingZoneResp => {
+        const assignedChannel = shippingZoneResp.channels.find(
+          channel => channel.name === randomChannel,
+        );
+        expect(assignedChannel).to.be.ok;
+      });
+    },
+  );
+  xit(
```
Comment (karola312) [inline cypress/e2e/configuration/channels/channels.js:127]: You can remove this test, not just disable it

**Review by karola312 — APPROVED** [review]
Remember to add this changes also to 3.5 branch.

**Pull request merged**